### PR TITLE
Designed and Developed home screen with new icons and features-Enhanc…

### PR DESCRIPTION
**Pull request by mujafar7**
…ed Home Screen UI

Screenshot_202310
![Screenshot_20231003_230602

Screenshot_20231003_231039
Screenshot_20231003_230611
Screenshot_20231003_230621
Screenshot_20231003_230604
Screenshot_20231003_230559
Screenshot_20231003_230555
Screenshot_20231003_230551
Screenshot_20231003_231055

### CODE REVIEW

File: src/pages/DisplayChapters.tsx [hunk 1]
```diff
@@ -0,0 +1,450 @@
...
+  const getCourseByUrl = localGradeMap?.courses.find(
+    (course) => courseDocId == course.docId
+  );
+
+  useEffect(() => {
+    init();
+  }, []);
+  useEffect(() => {
+    if (getCourseByUrl && !currentCourse) {
+      onCourseChanges(getCourseByUrl);
```
Comment (bhanugummana) [inline src/pages/DisplayChapters.tsx:74]: why do you need this in useEffect??

File: src/pages/DisplayChapters.tsx [hunk 2]
```diff
@@ -0,0 +1,450 @@
...
+    }
+  }, [getCourseByUrl]);
+  useEffect(() => {
+    console.log("chapters", currentCourse);
+    console.log("local grade map", localGradeMap);
+
+    getLocalGradeMap();
```
Comment (bhanugummana) [inline src/pages/DisplayChapters.tsx:81]: try to avoid using multiple useEffects

**Review by bhanugummana — CHANGES_REQUESTED** [review]

**Comment by mujafar7** [comment]
Screenshot_20231004_134052
Screenshot_20231004_134047

File: src/components/displaySubjects/SelectCourse.tsx [hunk 3]
```diff
@@ -1,37 +1,81 @@
...
+          </SplideSlide>
         );
       })}
-    </div>
+      {
+        <SplideSlide className="slide">
+          <div className="subject-button">
+            <div
+              className="course-icon"
+              style={{
```
Comment (Sutara-Skanda) [inline src/components/displaySubjects/SelectCourse.tsx:65]: move css style to selectCouse.css file

File: src/pages/Home.tsx [hunk 4]
```diff
@@ -104,14 +113,30 @@ const Home: FC = () => {
...
     urlOpenListenerEvent();
+    const urlParams = new URLSearchParams(location.search);
+    
+    if (!!urlParams.get("continue")) {
+      console.log("-------------line 126 current header", currentHeader);
+      setCurrentHeader(currentHeader);
+    }
   }, []);
 
+  useEffect(() => {
```
Comment (Sutara-Skanda) [inline src/pages/Home.tsx:133]: use single useEffect

**Review by Sutara-Skanda — CHANGES_REQUESTED** [review]

**Review by Sutara-Skanda — APPROVED** [review]

**Pull request merged**